AutoExp: false
ExpTime: 2000
Resolution: Normal
Gain: 200
AutoWB: false
HWTrigger: false
HWFrequency: 150
Record: false
FpsLimit: 0
RecordFps: 160
